manual/installation.tex

```diff
@@ -686,81 +686,82 @@ \subsection{Installing on NERSC Edison Cray XC30}
 
 \subsection{Installing on NERSC Cori, Haswell Partition, Cray XC40}
 Cori is a Cray XC40 with 16-core Intel "Haswell" nodes
-installed at NERSC. The build settings are similar to eos at OLCF, but
-a workround is needed for the newer Cray OS which does not include a static
-libXml2 library. A NERSC installed version and its dependencies are used:
+installed at NERSC.
 
 \begin{verbatim}
+module swap craype-haswell craype-mic-knl
+module unload cray-libsci  
+module load boost  
 module load cray-hdf5
 module load cmake
 module load fftw
 export FFTW_HOME=$FFTW_DIR/..
```

Comment:
do we still need fftw? MKL has provided FFT

Reply:
will check